features/step_definitions/godmin_steps.rb

```diff
@@ -15,8 +15,11 @@
   click_link_or_button 'Sign in'
 end
 
-Given(/^I visit the "([^"]*)" page$/) do |arg1|
-  pending # Write code here that turns the phrase above into concrete actions
+Given(/^I visit the "([^"]*)" page$/) do |path|
```

Comment:
Is godmin_steps the right place for this step?  It has nothing to do with godmin particularly, certainly not now you're trying to use it for the documentation test - or could that be why it fails, that this method is somehow "scoped" to the admin pages and /api is entirely outside of that?